recipes/python-pgplot/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+# Build and install giza from source (like CIBW_BEFORE_ALL)
+cd /tmp
+wget https://github.com/danieljprice/giza/archive/refs/tags/v1.4.2.tar.gz
+tar -xzf v1.4.2.tar.gz
+cd giza-1.4.2
+
+# Configure and build giza
+export CFLAGS="-fPIC"
+export CXXFLAGS="-fPIC"
+export LDFLAGS="-L${PREFIX}/lib"
+export CPPFLAGS="-I${PREFIX}/include"
```

Comment:
I am missing something, why are we building giza from source if it is available from conda-forge?